modules/jenkins_master/main.tf

```diff
@@ -0,0 +1,17 @@
+#
+# Creates Jenkins Infrastructure
+#
+
+variable "aws_key_pair" {}
+variable "tool_name"    {}
+
+resource "aws_instance" "jenkins_master" {
+  ami               = "ami-4bf3d731"
+  instance_type     = "t2.micro"
```

Comment:
This instance_type is too small. See https://liatrio.atlassian.net/wiki/spaces/PNCAP/pages/386596886/API+Fest+Pipeline+Server+Requirements